README.md

```diff
@@ -4,26 +4,75 @@ The official [shareable ESLint config](http://eslint.org/docs/developer-guide/sh
 
 ## Usage
 
----
-
 We export three ESLint configurations.
 
-### eslint-config-cesium
+### eslint-config-cesium default
+
+This config contains basic Cesium syntax and style config, from which `browser` and `node` extend. Extends `@eslint/js` recommended rules and `eslint-config/prettier` with additional rules.
+
+[`eslint.config.js`](https://eslint.org/docs/latest/use/configure/configuration-files)
+
+```js
+import configCesium from "eslint-config-cesium";
 
-This config contains basic Cesium syntax and style config, from which `browser` and `node` extend. Extends `eslint:recommended` and `prettier` with additional rules.
+exports default [
+    // Apply recommended rules to all files
+    configCesium.configs.recommended,
+    // Apply recommended rules to JS files with an override
+    {
+        files: ["**/*.js"],
+        rules: {
+            ...configCesium.configs.recommended.rules,
+            "no-unused-vars": "off"
+        } 
+    },
+]
+```
 
 ### eslint-config-cesium/browser
 
 For use in browser environments.
 
+[`eslint.config.js`](https://eslint.org/docs/latest/use/configure/configuration-files)
+
+```js
+import configCesium from "eslint-config-cesium";
+
+exports default [
+    // Apply recommended rules to all files
+    configCesium.configs.browser,
+    // Apply recommended rules to JS files with an override
+    {
+        files: ["**/*.js"],
```

Comment:
Is it normal for a ruleset expected to be used by others to list files at all? Or is that up to the caller?

Reply:
This is in the recommended ways to use it in other people's apps, not defining files within our config itself

Reply:
I may be misunderstanding, but the caller can apply the rules however they'd like. 

This was an illustrative example I stole from the [@eslint/js docs](https://www.npmjs.com/package/@eslint/js).

Reply:
Yeah, I was confused no worries.